src/js/engagement_view/src/components/reusableComponents/drawer.tsx

```diff
@@ -34,6 +35,21 @@ const DrawerHeader = styled("div")(({ theme }) => ({
 export const NavigationDrawer = () => {
     const classes = useStyles();
 
+    const location = window.location.href.split("/").slice().pop();
+
+    let index: number = 0;
+
+    location === "analyzers" ? (index = 0) : (index = 2);
+
```

Comment:
Okay, this I'd block the PR on. It's basically re-stating the "analyzers is 0" mapping in collapsibleDrawer.tsx, but obviously the code here is wrong (it's not always 2 if it's not analyzers)

Reply:
While I don't love using the window.location as a tracker of state, you could use it and the aforementioned 
```

const items: ListItem[] = [
  { label: "Analyzers", link: "/analyzers" },
  { label: "Engagements", link: "/engagements" },
];
```
to accurately figure out which index it currently is.

Reply:
Set active link with upgrade to react router!